### PR TITLE
Updating the views to use the new style jsoup tests from the ViewTest helper

### PR DESCRIPTION
**Pull request by joeyymoore**

### CODE REVIEW

File: test/views/OverviewPageViewSpec.scala [hunk 1]
```diff
@@ -39,306 +39,148 @@ class OverviewPageViewSpec extends AnyWordSpec with Matchers with GuiceOneAppPer
...
-        individualWithNoPriorData.select(task1DividendsP2Selector).text shouldBe task1NotStarted
-      }
+      lazy implicit val individualWithNoPriorData: Document = Jsoup.parse(individualWithNoPriorDataView.body)
 
-      s"has a link for $task1InterestsLinkText" which {
-        s"has the text of $task1InterestsLinkText" in {
-          individualWithNoPriorData.select(task1InterestsSelector).text shouldBe task1InterestsLinkText
-        }
+      linkCheck(vcBreadcrumb, vcBreadcrumbSelector, vcBreadcrumbUrl)
+      textOnPageCheck(startPageBreadcrumb, startPageBreadcrumbSelector)
```
Comment (ina-tu) [inline test/views/OverviewPageViewSpec.scala:100]: Would this be a linkCheck?

File: test/views/OverviewPageViewSpec.scala [hunk 2]
```diff
@@ -39,306 +39,148 @@ class OverviewPageViewSpec extends AnyWordSpec with Matchers with GuiceOneAppPer
...
-        agentWithNoPriorData.select(task1p1Selector).text shouldBe task1p1Agent
-      }
+      lazy implicit val agentWithNoPriorData: Document = Jsoup.parse(agentWithNoPriorDataView.body)
 
-      s"has a link for $task1DividendsLinkText" which {
-        s"has the text of $task1DividendsLinkText" in {
-          agentWithNoPriorData.select(task1DividendsSelector).text shouldBe task1DividendsLinkText
-        }
+      linkCheck(vcBreadcrumb, vcBreadcrumbSelector, vcAgentBreadcrumbUrl)
+      textOnPageCheck(startPageBreadcrumb, startPageBreadcrumbSelector)
```
Comment (ina-tu) [inline test/views/OverviewPageViewSpec.scala:129]: and here

File: test/views/OverviewPageViewSpec.scala (hunk 1 above)
Comment (joeyymoore) [reply test/views/OverviewPageViewSpec.scala]: updated test

File: test/views/OverviewPageViewSpec.scala (hunk 2 above)
Comment (joeyymoore) [reply test/views/OverviewPageViewSpec.scala]: updated test

**Review by ina-tu — APPROVED** [review]

**Pull request merged**